File: client/src/autogenerated/application/activities/input/activity-list-input-model.ts
import { PagingParams } from '../../common/pagination/paging-params';
import { ActivityAttendingFilterType } from '../enums/activity-attending-filter-type.enum';

export class ActivityListInputModel extends PagingParams {
    attending!: ActivityAttendingFilterType;
    startDate?: Date;
}

File: Reactivities.Server/Application/Activities/Models/Input/ActivityListInputModel.cs
using System;
using Application.Activities.Models.Enums;
using Application.Common.Models.Pagination;

namespace Application.Activities.Models.Input;

public class ActivityListInputModel : PagingParams
{
    public ActivityAttendingFilterType Attending { get; set; }

    public DateTime? StartDate { get; set; } = null;
}